_gsocblogs/2022/blog_Acts_TirthankarMazumder.md

```diff
@@ -0,0 +1,55 @@
+---
+project: Acts 
+title: Vectorized Linear Algebra Implementation for Acts
+author: Tirthankar Mazumder
+avatar: https://github.com/wermos.png
+date: 3.10.2022
+year: 2022
+layout: blog_post
+logo: ACTSlogo.gif
+intro: |
+  This blog is about my experiences and what I learned during my GSoC project, which involved adding a new math backend to algebra-plugins, to make Acts faster.
+---
+
+Hi everyone! Since starting this year, blogs are mandatory for all CERN-HSF contributors, I decided to take this time to make my own blog. I have written about my journey extensively there. Do [give it a visit](https://wermos.github.io/blog/)!
+
+I have broken down the midterm project report into multiple blog posts:
+
+The [first blog post](https://wermos.github.io/blog/gsoc/gsoc-first-blog-post/) discusses what GSoC is in general, for those who might just stumble upon my blog by happenstance.
+
+The [second blog post](https://wermos.github.io/blog/gsoc/gsoc-the-details-of-my-project/) delves into the technical details about what my project actually is, what repository I would be working with, mentions our weekly meeting schedule, and describes my three mentors.
+
+The [third blog post](https://wermos.github.io/blog/gsoc/gsoc-the-work-so-far/) finishes up the trilogy by exhaustively discussing the work up till now, including benchmarking data and conclusions, and the current direction of the work.
+
+That being said, I will also share a short report of the work of the work I am doing, and the link to the things I have done so far below:
+
+## My progress
+
+### Fast 5×5
+One of the main things we need, to efficiently figure out what to improve first, is a comprehensive benchmarking suite for the existing math backends. To that end, I added Google Benchmark to the [`algebra-plugins` repository](https://github.com/acts-project/algebra-plugins), in [PR #69](https://github.com/acts-project/algebra-plugins/pull/69). Writing those benchmarks is still a work-in-progress.
+
+Then, my mentors and I spent a lot of time testing out 3 possible math backend alternatives (the custom-made `fast5x5`, `blaze`, and `Fastor` in a repository of mine called [Fast 5×5](https://github.com/wermos/Fast5x5) and comparing their performance to that of `Eigen`. This repository was something I forked from some [prior work done on this front](https://gitlab.in2p3.fr/CodeursIntensifs/Fast5x5).
```

Comment:
Maybe, you can also provide some links to the blaze, Fastor and Eigen projects?

Reply:
Done.